src/methods/clientMethods.mjs

```diff
@@ -137,17 +138,58 @@ export async function getOptionsForFilters(params) {
 }
 
 export async function getSeance(params) {
-  const { seanceId } = params;
+  const { seanceId, userId } = params;
   const seance = await Seance.findById(seanceId);
   const movieTheater = await MovieTheater.findOne(
     { 'halls._id': seance.hallId },
     { halls: { $elemMatch: { _id: seance.hallId } } }
   ).select('-seances');
-  console.log('seance', seance);
-  console.log('movieTheater', movieTheater);
-
+  const nowTime = new Date();
+  nowTime.toISOString();
+  const blockedSeatsQuery = BlockedSeats.find({ seanceId }).where({ expireAt: { $gte: nowTime } });
+  const blockedSeats = await blockedSeatsQuery;
+  let blockedSeatsByUser = [];
+  if (userId) {
+    blockedSeatsByUser = await blockedSeatsQuery.where({ userId }).sort('expireAt');
+    console.log(blockedSeatsByUser);
+  }
   return {
     seance,
     cinemaInfo: movieTheater,
+    blockedSeats,
+    blockedSeatsByUser,
   };
 }
+
+export async function toBlockSeat(params) {
+  const { seanceId, seat, row, userId } = params;
+
+  try {
+    const blockedSeat = await BlockedSeats.findOne({ seanceId, seat, row });
+    if (blockedSeat) {
+      return { success: false, message: 'seat is already blocked' };
```

Comment:
You can create sort of dictionary of such error nmessgaes in separate file

Reply:
fixed